training/computational-chemistry/bio3danalysis/MD_Analysis_using_Bio3D.ga

```diff
@@ -0,0 +1 @@
__page__\": null, \"__rerun_remap_job_id__\": null, \"method\": \"\\\"false\\\"\", \"pdbin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"pca\": \"{\\\"__current_case__\\\": 0, \\\"sele\\\": \\\"calpha\\\"}\", \"chromInfo\": \"\\\"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\", \"dcdin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", "id": 3, "tool_shed_repository": {"owner": "chemteam", "changeset_revision": "f56fb2762abb", "name": "bio3d_pca", "tool_shed": "toolshed.g2.bx.psu.edu"}, "uuid": "be585280-9e47-47bb-b4ea-d9b8375c8204", "errors": null, "name": "PCA", "post_job_actions": {"HideDatasetActionpc1_rmsf": {"output_name": "pc1_rmsf", "action_type": "HideDatasetAction", "action_arguments": {}}, "HideDatasetActionpca_cluster": {"output_name": "pca_cluster", "action_type": "HideDatasetAction", "action_arguments": {}}, "HideDatasetActionpca_plot": {"output_name": "pca_plot", "action_type": "HideDatasetAction", "action_arguments": {}}}, "label": null, "inputs": [{"name": "pdbin", "description": "runtime parameter for tool PCA"}, {"name": "dcdin", "description": "runtime parameter for tool PCA"}], "position": {"top": 104, "left": 667}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/chemteam/bio3d_pca/bio3d_pca/2.3", "type": "tool"}, "4": {"tool_id": "toolshed.g2.bx.psu.edu/repos/chemteam/bio3d_rmsd/bio3d_rmsd/2.3", "tool_version": "2.3", "outputs": [{"type": "tabular", "name": "output"}, {"type": "png", "name": "rmsd_plot"}, {"type": "png", "name": "rmsd_hist_plot"}], "workflow_outputs": [{"output_name": "output", "uuid": "f7bae0e8-b065-4ffb-9b9c-5348d4f27fc9", "label": "rmsd_tabular_output"}], "input_connections": {"pdbin": {"output_name": "output", "id": 0}, "dcdin": {"output_name": "output", "id": 1}}, "tool_state": "{\"__page__\": null, \"rmsd\": \"{\\\"__current_case__\\\": 0, \\\"sele\\\": \\\"calpha\\\"}\", \"__rerun_remap_job_id__\": null, \"pdbin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"chromInfo\": \"\\\"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\", \"dcdin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", "id": 4, "tool_shed_repository": {"owner": "chemteam", "changeset_revision": "75fd897bd85d", "name": "bio3d_rmsd", "tool_shed": "toolshed.g2.bx.psu.edu"}, "uuid": "8088ee87-34b5-4499-9419-4fcceb2f2a4a", "errors": null, "name": "RMSD Analysis", "post_job_actions": {"HideDatasetActionrmsd_hist_plot": {"output_name": "rmsd_hist_plot", "action_type": "HideDatasetAction", "action_arguments": {}}, "HideDatasetActionrmsd_plot": {"output_name": "rmsd_plot", "action_type": "HideDatasetAction", "action_arguments": {}}}, "label": null, "inputs": [{"name": "pdbin", "description": "runtime parameter for tool RMSD Analysis"}, {"name": "dcdin", "description": "runtime parameter for tool RMSD Analysis"}], "position": {"top": 403, "left": 735}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/chemteam/bio3d_rmsd/bio3d_rmsd/2.3", "type": "tool"}}, "annotation": "", "a_galaxy_workflow": "true"}
```

Comment:
Mh, the other workflow was more readable :(

Reply:
will pretty format with jq and I am still fixing that yml, running in planemo now

Reply:
No problem @chrisbarnettster, just ping us when it's ready!